README.md

```diff
@@ -28,7 +28,12 @@ Wrap your application with the `<Pond>` to use you fish everywhere in the code.
 
 ```js
 ReactDOM.render(
-  <Pond>
+  <Pond
+    onError={() => {
+      setTimeout(() => location.reload(), 5000)
+      return <div>Connection to Actyx rejected: {JSON.stringify(e)}. Next reconnect in 5 seconds.</div>
```

Comment:
would be better to show the current time here so that it is obvious if the page keeps reloading every 5sec

Reply:
I hope it is clear that this should not be used in factory apps. (UX designer would not survive that :-) )

Anny better way would be much more code. I try to avoid that.

Reply:
How about making that the default for when `onError` is not provided? Then the default could be described with its reasons in the docs, allowing users to come to the right conclusions.